tools/clang/test/lit.cfg

```diff
@@ -269,6 +269,41 @@ config.substitutions.append( ('%itanium_abi_triple', makeItaniumABITriple(config
 config.substitutions.append( ('%ms_abi_triple', makeMSABITriple(config.target_triple)) )
 config.substitutions.append( ('%dxc', lit.util.which('dxc', llvm_tools_dir)) )
 
+bin_dir = os.path.join(config.llvm_obj_root, lit_config.params['build_mode'], 'bin')
+config.environment['PATH'] = str.format('{};{}',config.environment['PATH'], bin_dir)
+
+
+# Displaying the parent directory of the script
+cur_dir = os.path.dirname(__file__)
+
+config.substitutions.append( ('%batch',
+                            'dxc_batch' ))
```

Comment:
Using `lit.util.which` allows you to leverage lit finding the tool. That might be a more resilient approach.